README.md

```diff
@@ -11,23 +11,56 @@ This guide demonstrates how to use the CellProfiler Python API to analyze data s
 
 This repository contains documentation and notebooks.
 
-To run the notebooks, you can either [run on mybinder.org](https://mybinder.org/v2/gh/ome/omero-guide-cellprofiler/master?filepath=notebooks) or build locally with [repo2docker](https://repo2docker.readthedocs.io/).
+## Run the notebooks
 
+### Running on cloud resources
 
-To build locally:
+[![Binder](https://mybinder.org/v2/gh/ome/omero-guide-cellprofiler/master?filepath=notebooks)
 
- * Install [Docker](https://www.docker.com/) if required
- * Create a virtual environment and install repo2docker from PyPI.
- * Clone this repository
- * Run  ``repo2docker``. 
- * Depending on the permissions, you might have to run the command as an admin
+The OMERO server used will need to have [websockets support](https://docs.openmicroscopy.org/omero/latest/sysadmins/websockets.html) enabled.
 
-```
-pip install jupyter-repo2docker
-git clone https://github.com/ome/omero-guide-cellprofiler.git
-cd omero-guide-cellprofiler
-repo2docker .
-```
+### Running in Docker
+
+Alternatively, if you have Docker installed, you can use the [repo2docker](https://repo2docker.readthedocs.io/en/latest/)
+tool to run this repository as a local Docker instance:
+
+
+    $ git clone https://github.com/ome/omero-guide-cellprofiler
+    $ cd omero-guide-cellprofiler
+    $ repo2docker .
+
+### Running locally
+
+Finally, if you would like to install the necessary requirements locally,
+we suggest using conda.
+
+Then, create the environment:
+
+    $ git clone https://github.com/ome/omero-guide-cellprofiler
+    $ cd omero-guide-cellprofiler
+    $ conda env create -n omero-guide-cellprofiler -f binder/environment.yml
+
+and activate the newly created environment:
+
+    $ conda activate omero-guide-cellprofiler
+
+The following steps are only required if you want to run the notebooks
+
+* If you have Anaconda installed:
+  * Start Jupyter from the Anaconda-navigator
+  * In the conda environment, run ``conda install ipykernel``
+  * To register the environment, run ``python -m ipykernel install --user --name omero-guide-cellprofiler``
+  * Select the notebook you wish to run and select the ``Kernel>Change kernel>Python [conda env:omero-guide-cellprofiler]``
```

Comment:
I do not have a menu item 
``Python [conda env:omero-guide-cellprofiler]``

Instead, I have just ``omero-guide-cellprofiler`` as a menu item (I call it ``test-jm-cp``) - this is a corresponding conda env naming :)


![Screenshot 2023-05-02 at 12 34 59](https://user-images.githubusercontent.com/2478303/235656087-ba2c7126-e498-4478-8b2d-2ee21deaa952.png)